### PR TITLE
Remove traditional client VM from Uyuni CI

### PR DESCRIPTION
**Pull request by srbarrios**
As you are aware, we will deprecate traditional client in 4.3, but we will support it until next version, so 4.4.
We must still testing it for SUMA 4.3 CI, but I make a proposal here to remove it from our Uyuni CI.

If we remove it from here, our test framework will automatically not run any Cucumber Feature that needs this VM.
Currently we have more than 20 features using it.

I estimate that not deploying this VM plus not running the features related to a traditional client, we can gain around 2 hours in the total execution time of a test suite.

### CODE REVIEW

**Review by Bischoff — COMMENTED** [review]
testing strategy side: as it reduces the test coverage, I think it needs approval from Nanuk and Abid. It would also be nice to get opinion from Julio and others.

**Comment by srbarrios** [comment]
The PR was not ready to share it yet ;) I will do once I have results that prove it's worth to remove it from Uyuni CI.

**Review by juliogonzalez — APPROVED** [review]

**Review by nodeg — APPROVED** [review]

**Comment by srbarrios** [comment]
Blocked until we give the approval for 4.3.1 and Uyuni submissions

**Pull request merged**
